### ISSUE TITLE
Naming convention

### ISSUE THREAD

**Issue by teamdisc** [galavebalaji/travelbooks#38]
In [Swift API guideline](https://swift.org/documentation/api-design-guidelines), there's a convention for function naming. Use verb when for mutating method and v.3/v.ing for non-mutating method. 

For example, I think in our case here;
```swift
func rounded(_ cornerRadius: CGFloat, withBorder: Bool = true, borderColor: UIColor, borderWidth: CGFloat = 0) 
```
it might make more sense to use just `round` instead since it actually mutates its own layer and properties. What do you think? 🤔

**Comment by galavebalaji** [comment]
Agreed. It makes sense to have a ```round``` instead of ```rounded```
Thanks.
